src/Microsoft.DotNet.XHarness.CLI/Commands/XHarnessHelpCommand.cs

```diff
@@ -50,8 +50,10 @@ public override int Invoke(IEnumerable<string> arguments)
                     PrintCommandHelp(new AndroidCommandSet(), subCommand);
                     break;
                 case "apple":
+#if !DEBUG
```

Comment:
probably it should be `DEBUG` here?

Reply:
We want to allow this command in DEBUG - so `#if RELEASE` require to be on OSX.